src/libprojectM/projectM.cpp

```diff
@@ -837,20 +844,26 @@ void projectM::switchPreset(const bool hardCut) {
     }
 
     populatePresetMenu();
+
+    return result;
```

Comment:
Not really clear to me why returning a string instead of bool.

Reply:
Yeah I also don't know why I did this. Changing it to bool.

Reply:
Ah I remember. It was to keep the return type consistent with the other overload. Why do we have two overloads for this function anyways? Seems like the overload taking `unique_ptr<Preset>&` should be named `switchToPreset`, and the other overload should be called `startTransition` or something like this.